src/lib.rs

```diff
@@ -281,3 +274,89 @@ pub unsafe extern "C" fn wgpuSurfaceGetPreferredFormat(
     };
     return preferred_format;
 }
+
+struct DeviceCallback<T> {
+    callback: T,
+    userdata: *mut std::os::raw::c_void,
+}
+
+type UncapturedErrorCallback = DeviceCallback<native::WGPUErrorCallback>;
+type DeviceLostCallback = DeviceCallback<native::WGPUDeviceLostCallback>;
+
+unsafe impl<T> Send for DeviceCallback<T> {}
+
+lazy_static::lazy_static! {
+    static ref UNCAPTURED_ERROR_CALLBACKS: Arc<Mutex<HashMap<id::DeviceId, UncapturedErrorCallback>>> = Arc::new(Mutex::new(HashMap::new()));
```

Comment:
First, I don't think `Arc` is needed here.

Secondly, can we merge them into a single structure?
```rust
Mutex<Callbacks>
struct Callbacks {
  uncaptured_errors: HashMap<id::DeviceId, UncapturedErrorCallback>,
  device_lost: HashMap<id::DeviceId, DeviceLostCallback>,
}
```

Reply:
`Mutex<Callbacks>` doesn't compile, but I collected them in a `static ref CALLBACKS: Mutex<Callbacks>`

Reply:
yes, that's what I thought :)